Gemfile

```diff
@@ -62,3 +62,8 @@ gem "haml", "~> 5.1"
 gem 'haml-rails', '~> 2.0', '>= 2.0.1'
 # Use i18n_generators to generate model attribute names automatically
 gem 'i18n_generators', '~> 2.2'
+#use omniauth to integrate AuthSCH
+gem 'omniauth'
+gem 'omniauth-oauth2'
+
+gem 'dotenv-rails', groups: [:development, :test]
```

Comment:
Why do we exclude the production environment?

Reply:
Or why do we set development and test?

Reply:
Well the github page of the dotenv gem said:
> dotenv was originally created to load configuration variables into ENV in development. There are typically better ways to manage configuration in production environments... 

I put the enviroment variables in the `docker-compose.yml` file, but if we want we can store all aour env variables in a dotenv file and use a `.env.production` file for the production variables.

Reply:
Good point! 
We should keep it as is.